db/db_test.go

```diff
@@ -133,7 +133,7 @@ var _ = Describe("Database operation test", func() {
 			dataStore, err = dbutil.ConnectDB(dbType, conn, db.DefaultMaxOpenConn, options.Default())
 			Expect(err).ToNot(HaveOccurred())
 
-			for _, s := range manager.Schemas() {
+			for _, s := range manager.OrderedSchemas() {
```

Comment:
Actually, this is the biggest problem why this random failure happened. MySQL strictly checks table dependencies, so `OrderedSchemas` function is used in gohan server for this purpose. However, this test was not maintained for MySQL, so table creations had been randomly happened.